src/actions/AuthActions.js

```diff
@@ -1,67 +1,105 @@
-import firebase from 'firebase';
+import {AsyncStorage} from 'react-native';
+import {Facebook} from 'expo';
+import axios from 'axios';
+
 import _ from 'lodash';
 
 import {
-    EMAIL_CHANGED,
-    PASSWORD_CHANGED,
-    LOGIN_AUTH_ERROR,
-    LOGIN_SUCCESS
+    APP_READY,
+    FACEBOOK_LOGIN_SUCCESS,
+    FACEBOOK_LOGIN_FAILURE,
+    FACEBOOK_LOGOUT_COMPLETE,
+    GUEST_TOKEN_SET
 } from './types';
 
-export const emailChanged = (text) => {
-    return {
-        type: EMAIL_CHANGED,
-        payload: text
-    };
-};
+import {FACEBOOK_AUTH} from '../../env';
+import {tokens} from '../constants';
 
-export const passwordChanged = (text) => {
-    return {
-        type: PASSWORD_CHANGED,
-        payload: text
```

Comment:
Removed `passwordChanged` as we're no longer allowing login with email/password.